blog/_posts/2017-06-30-collections-0.2.0.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: Redesigned collections v0.2.0
+---
+
+We are happy to announce the
+[0.2.0 release](https://index.scala-lang.org/scala/collection-strawman/collection-strawman/0.2.0)
+of the redesigned collections!
+
+You can read more about our effort to redesign the standard collections in this previous
+[blog post](http://www.scala-lang.org/blog/2017/02/28/collections-rework.html).
+
+## Current status
+
+The goal of this release is to provide a drop-in replacement of the current collections
+for most use cases: the main collection types (`Seq`, `Set` and `Map`, mutable or
+immutable, sorted or not sorted) are implemented
+and support the most common operations. You can see a detailed list of the current
+implementation status
+[here](https://github.com/scala/collection-strawman#implemented-collection-types).
+
+To use it in your project, add the following dependency:
```

Comment:
Why not say instead, "To experiment with the latest milestone of the new collections library, just add the following dependency:"

(You don't necessarily want people using this library in production or anything, right? "Use in your project" makes it sound like you do)